### DISCUSSION TITLE
Cannot run mockoon-cli

### DISCUSSION THREAD

**Discussion by LukMas** [mockoon/mockoon#1142] [Help] [answered]
Hi!

I'm trying to user **mockoon-cli** on my **kubuntu** but I can't get it working. 
I've installed the _snap_ of node (because, it looks better than the native version). So: 
`sudo snap install node --classic`
Then I've restarted the system and now I get both `npm` and `node` in their latest versions.
Then I've run 
`npm install -g @mockoon/cli`
As suggested on [this page](https://github.com/mockoon/mockoon/tree/main/packages/cli#compatibility)

No way. I can't get it working. 
Every time I run 
`mockoon-cli `
I get the **command not found** error. 

Could someone help me with this, please? 

Note: I'm sure that the problem is not the snap package of node, every other thing I do with node works. 

-------------- EDIT ---------------

The problem has definitely something to do with ***buntu**. On the other system that runs on Fedora it works without a problem.

**Comment by 255kb** [comment, marked as answer]
Hi, 
I suspect that the command wasn't added to the PATH. It could be due to a broken Node.js installation, lack of permissions, etc.
It's a bit out of Mockoon's scope, but I suggest that you check NPM's global modules folder (possibly in `/usr/local/lib/node_modules`) for the mockoon-cli command. If this folder is not in the PATH variable you may have a Node.js installation issue.

**Reply by LukMas** [reply]
Hi, 

Thanks for the reply. I was really busy at work and I've forgotten to write back. 
Yes, the thing has nothing to do with Mockoon. Indeed the content of `node_modules` was empty and I was not able to find where the `mockoon_cli` was installed. 

It has to to with **Kubuntu** and the way that in general Ubuntu (Debian too?) manages `node` and `npm`. I've solved the problem installing the snap version of node, just to find that then another piece of software doesn't work. 
I've lost all the morning trying to find out way the other software now doesn't work, but before yes. On the other side, not mockoon-cli works, while before not.
